### PR TITLE
[Higher-Level Review static page] Add online wizard 

### PR DESCRIPTION
**Pull request by ncksllvn**
I added this widget into the page based on the [mockup](https://vsateams.invisionapp.com/share/W5U21BDFTQK#/screens/388784208/comments) but I only considered the "online" section, so there may be updates I missed.

## Page to edit
url: /decision-reviews/higher-level-review/

## Origin of request (internal/stakeholder/user feedback)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/4139

## Description of what's needed (edits/link changes/additions)
Add the online section w/wizard

### CODE REVIEW

File: pages/decision-reviews/higher-level-review.md [hunk 1]
```diff
@@ -51,6 +51,35 @@ If you request an informal conference with a senior reviewer, they’ll call the
...
+      Sorry, this is taking longer than expected.
+    </span>
+  </div>
+  <span class="static-widget-content vads-u-display--none" aria-hidden="true">
+  </span>
+    <div class="usa-alert usa-alert-error sip-application-error vads-u-display--none" aria-hidden="true">
+      <div class="usa-alert-body">
+        <strong>We’re sorry. Something went wrong when we tried to load your saved application.</strong><br>Please try refreshing your browser in a few minutes.
+      </div>
+    </div>
```
Comment (Mottie) [inline pages/decision-reviews/higher-level-review.md]: The indention of this div block should line up with the span above. Other than that, it looks good!

**Comment by ncksllvn** [comment]
Guessing I can close this @Mottie ?

File: pages/decision-reviews/higher-level-review.md [hunk 2]
```diff
@@ -104,14 +139,35 @@ If you have evidence to submit, please select another review option.
...
+**In person**
+<br>
+Bring your completed form to a regional office.
+<br>
+[Find a regional office near you](/find-locations/)
+<br>
+
+**By fax**
+<br>
+Fax your completed form and any supporting documents to 844-531-7818.
```
Comment (Mottie) [inline pages/decision-reviews/higher-level-review.md:164]: I don't know what the state of faxing is right now... we were all set to remove it - see https://github.com/department-of-veterans-affairs/vets-website/pull/15203 - but then was told to hold off.

File: pages/decision-reviews/higher-level-review.md [hunk 3]
```diff
@@ -104,14 +139,35 @@ If you have evidence to submit, please select another review option.
 
 <li class="process-step list-four">
 
-### Apply
+### Mail, fax, or drop off the form
 
 Fill out the Decision Review Request: Higher-Level Review (VA Form 20-0996). <br>
 <a href="https://www.vba.va.gov/pubs/forms/VBA-20-0996-ARE.pdf">Download VA Form 20-0996 (PDF)</a>
 
+You can also ask for a copy of this form at a regional office, or by calling the VA toll-free hotline, 800-827-1000, Monday through Friday, 8:00 a.m. to 9:00 p.m.
```
Comment (Mottie) [inline pages/decision-reviews/higher-level-review.md:147]: I'm not sure if this applies here, but in HLR's footer, we're using the MYVA411 number which has the hours of 24/7 - see https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/996/content/GetFormHelp.jsx#L11

**Comment by ncksllvn** [comment]
`/decision-reviews/higher-level-review/` is in Drupal now, so this PR isn't relevant any longer.

**Comment by ncksllvn** [comment]
Slack thread to fully close the loop, https://dsva.slack.com/archives/C0G5ZFK5J/p1614019985006300.

**Pull request closed without merging**